### PR TITLE
i/6131: Table and table cell property form fields should be validated

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Table and table cell property form fields should be validated. Closes ckeditor/ckeditor5#6131.

---

### Additional information

image

Requires:
- [x] [A PR in theme-lark that brings styles for rich error messages](https://github.com/ckeditor/ckeditor5-theme-lark/pull/265).

### CODE REVIEW

File: src/ui/utils.js [hunk 1]
```diff
@@ -110,6 +113,60 @@ export function getBorderStyleLabels( t ) {
...
+/**
+ * Returns a localized error string that can be displayed next to length (padding, border width)
+ * fields that have an invalid value.
+ *
+ * @param {module:utils/locale~Locale#t} t The "t" function provided by the editor
+ * that is used to localize strings.
+ * @returns {String}
+ */
+export function getLocalizedLengthErrorText( t ) {
+	return t( 'The value is invalid. Try "10px" or "2em" or simply "2".' );
```
Comment (oleq) [inline src/ui/utils.js]: @jodator I assumed here https://github.com/ckeditor/ckeditor5/issues/6098 will be merged soon.

Comment (jodator) [reply src/ui/utils.js]: :crossed_fingers: hopefully - looking for a reviewers :P

Comment (jodator) [reply src/ui/utils.js]: Oh - this one should be done on Monday. I though this one was about normalizing value passed to a command.

File: tests/tablecellproperties/tablecellpropertiesui.js [hunk 2]
```diff
@@ -215,22 +218,187 @@ describe( 'table cell properties', () => {
...
+						sinon.assert.calledWithExactly( spy, 'tableCellBorderColor', { value: '#FFAAFF', batch: 'foo' } );
+					} );
+
+					it( 'should display an error message if value is invalid', () => {
+						const spy = testUtils.sinon.stub( editor, 'execute' );
+
+						// First, let's pass an invalid value and check what happens.
+						tableCellPropertiesView.borderColor = '42';
+
+						clock.tick( 500 );
```
Comment (jodator) [inline tests/tablecellproperties/tablecellpropertiesui.js]: :heart:

Comment (oleq) [reply tests/tablecellproperties/tablecellpropertiesui.js]: Sure

**Pull request merged**